### PR TITLE
Allow post_navigation frontmatter override

### PR DESCRIPTION
**Pull request by joshuaulrich**
Now the 'post_navigation' parameter can be overridden in a page's frontmatter, which is consistent with how the 'authorbox' parameter is handled.

This allowed me to put pages in the main menu that looked more like single pages, rather than blog posts.

### CODE REVIEW

**Comment by joshuaulrich** [comment]
Thanks for this great theme!  I hope you agree this was sufficiently simple enough that an issue wasn't needed.  Let me know if you'd like any more changes (e.g. an addition to the README).

**Comment by Vimux** [comment]
@joshuaulrich,

> Now the 'post_navigation' parameter can be overridden in a page's frontmatter, which is consistent with how the 'authorbox' parameter is handled.

Agreed, but I'm not happy with the current "post_navigation" partial name (not only as param name), and I want to replace this with one word. Maybe "pager" or something else? WDYT?

It is better to change the partial name and all corresponding stuff right before this PR so as not to make breaking front matter changes later. I should be able to make the necessary changes later this week.

**Comment by joshuaulrich** [comment]
Personally, I would not change the name unless there's a very compelling reason.  I would prefer "paginator", but I assume that would conflict with Hugo's built-in parameter.  You know Hugo templates better than me though, I'm sure.  Sorry I don't have more helpful thoughts/suggestions.

**Comment by joshuaulrich** [comment]
@Vimux, Git thinks you deleted the `post_nav.html` partial and added the `pager.html` partial, rather than renaming the file.  So that creates a merge conflict with my parameter change.  How would you like me to handle that?  I can cherry-pick your commit into my branch, or I can merge your upstream/master into my branch, or I can leave the merge conflict for you to resolve when you merge this PR.  What do you prefer?

**Comment by Vimux** [comment]
> ...I can merge your upstream/master into my branch

This

**Comment by joshuaulrich** [comment]
Okay, that's what I did, and I just pushed those changes.  I tested them and they worked for me.

File: README.md [hunk 1]
```diff
@@ -153,6 +153,7 @@ thumbnail: "img/placeholder.jpg" # Thumbnail image
 lead: "Example lead - highlighted near the title" # Lead text
 comments: false # Enable Disqus comments for specific page
 authorbox: true # Enable authorbox for specific page
+pager: false # Disable pager for specific page
```
Comment (Vimux) [inline README.md:156]:
```suggestion
pager: true # Enable pager navigation (prev/next links) for specific page
```

It looks more consistent with `authorbox` param.

File: layouts/partials/post_nav.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{{- if .Param "post_navigation" }}
```
Comment (Vimux) [inline layouts/partials/post_nav.html:1]: Delete this file

Comment (joshuaulrich) [reply layouts/partials/post_nav.html:1]: Sorry about that. I must have missed that during the merge...

**Comment by Vimux** [comment]
@joshuaulrich,

Thank you!

**Pull request merged**